src/lib/pages/validator-details/components/validator-overview/VotingPowerOverview.tsx

```diff
@@ -1,21 +1,32 @@
-import { Flex, Heading, Text } from "@chakra-ui/react";
+import { Flex, Grid, Heading, Text } from "@chakra-ui/react";
+import type { BigSource } from "big.js";
+import type Big from "big.js";
 
+import { TokenImageRender } from "lib/components/token";
 import { ValueWithIcon } from "lib/components/ValueWithIcon";
-
-// TODO remove mockup data
+import { getUndefinedTokenIcon } from "lib/pages/pools/utils";
+import type { AssetInfos, Option, Token, U, USD } from "lib/types";
+import {
+  calculateAssetValue,
+  divWithDefault,
+  formatPrettyPercent,
+  formatPrice,
+  formatUTokenWithPrecision,
+  toToken,
+} from "lib/utils";
 
 const VotingPowerDetail = ({
   label,
   percent,
   amount,
-  denom,
+  denom = "",
   value,
 }: {
   label: string;
   percent: string;
   amount: string;
-  denom: string;
-  value: string;
+  denom: Option<string>;
```

Comment:
Since, @evilpeach recommended to move percent & amount formatted/calculation to this component. We need to passed `assetInfo` which already contain `symbol`. Therefore, I will remove `symbol prop` and use the `symbol` value that contained inside `assetInfo` instead na.